examples/src/main/scala/mhtml/examples/HelloWorld.scala

```diff
@@ -1,4 +1,5 @@
 package mhtml.examples
+import mhtml._
```

Comment:
I know, it's a shame. How about we customize the `implicitNotFound` message in the `scala-xml-light` project to the previous message:

```
Cannot embed value of type ${T} in xml element. Here's what you can try:
1. `import mhtml._`
2. Use `${T}.toString`
3. Find which part of ${T} is disallowed. Only the following types are allowed
  - String, Int, Long Double, Float
  - xml.Node, Iterable[xml.Node]
  - mhtml.Var[T], mhtml.Rx[T] where T can be embedded in html
```

This means only the error message (which is easy to move) is tied to mhtml while scala-xml.light remains a separate module.